src/apps/bridge/aanderaaController.cpp

```diff
@@ -40,14 +45,16 @@ typedef struct aanderaaControllerCtx {
   bool _initialized;
   TimerHandle_t _sample_timer;
   BridgePowerController *_bridge_power_controller;
+  cfg::Configuration *_usr_cfg;
 } aanderaaControllerCtx_t;
 
 static aanderaaControllerCtx_t _ctx;
 
+static constexpr uint32_t DEFAULT_CURRENT_AGG_PERIOD_MIN = 3;
+static constexpr uint32_t CURRENT_SAMPLE_PERIOD_MS = 2000; // Aanderaa "interval" config setting
```

Comment:
This is hardcoded for now and handled & set on the Aanderaa by manufacturing. 
We can add Aanderaa configuration functionality later